app/components/user-navigation/template.hbs

```diff
@@ -0,0 +1,44 @@
+  {{#if isLoggedIn}}
+    <div class="user-nav-logged-in">
+      {{#click-outside action=(action 'closePopup')}}
+        <button {{action 'togglePopup'}}>
+          <div class="user-nav-avatar">
+          {{#if user.picture}}
+            <img src={{user.picture}}>
+          {{else}}
+            {{wnyc-svg icon="user"}}
+          {{/if}}
+          </div>
+          <span class="user-nav-greeting">Hi, {{or user.givenName 'You'}}</span>
+          <span class="user-nav-triangle"></span>
+        </button>
+        {{#if isPopupOpen}}
+          <div class="user-nav-popup-pointer"></div>
```

Comment:
The arrow points down from the balloon on that component. We need an arrow that points up from the balloon. That means the elements in a different order and with different CSS.

 I would update the template and css to add another direction if that was the only issue.

But the other issue is currently the component takes some text and an icon name (from wnyc-svg) to define its button (the yield is used for the popup contents). There are a few better ways to do this but they all take work.

Since we don't need smart horizontal positioning here I just used a simple toggle.